grpc/thread_pool.cpp

```diff
@@ -0,0 +1,38 @@
+#include "thread_pool.hpp"
+
+#include "job.hpp"
+
+namespace ioremap { namespace grpc {
+
+thread_pool_t::thread_pool_t() = default;
```

Comment:
1. Code of `thread_pool_t::make` must be in constructor. We must create thread pool as:
`thread_pool = std::make_unique<thread_pool_t>(cq, size)`. Prefer pattern RAII: https://habr.com/en/sandbox/21603/

2. Let's rename `thread_pool_t` to something like `completion_thread_pool_t`, to avoid name- or meaning- collision in the future. At least in namespace `grpc` can be another async tasks, not only completion - so you've reserved too common name.

Reply:
f24a094